src/documentation/sdks/python-streaming.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: section
+title: "Apache Beam Python Streaming Pipelines"
+section_menu: section-menu/sdks.html
+permalink: /documentation/sdks/python-streaming/
+---
+
+# Python Streaming Pipelines
+
+Apache Beam SDK for Python supports streaming pipeline execution as of version
+TBD. Currently, two Beam runners support Python streaming execution:
```

Comment:
2 beam runners are the only two beam python runners anyway. I do not know if this is worth mentioning.